client_admin/pulp/client/admin/tasks.py

```diff
@@ -31,6 +32,15 @@ class BaseTasksSection(PulpCliSection):
     to provide consistent functionality for a subset of tasks.
     """
 
+    all_flag = PulpCliFlag('--all',  _('if specified, all tasks in all states are shown'),
+                           aliases=['-a'])
+    state_option = PulpCliOption('--state',
+                                 _('comma-separated list of tasks states desired to be '
+                                   'shown. Example: "runnining,waiting,canceled,successful,failed". '
```

Comment:
also, just confirm that we spell it "canceled". One "l" or two has been the source of much confusion in the past, and I can never remember which one we ended up choosing.